### ISSUE TITLE
pkg-config file contains invalid relative paths

### ISSUE THREAD

**Issue by mgorny** [libgit2/libgit2#5453]
### Reproduction steps

Build without passing `INCLUDE_INSTALL_DIR` or `LIB_INSTALL_DIR` to CMake.

### Expected behavior

pkg-config file containing correct `-L` and `-I` arguments.

### Actual behavior

pkg-config file has:

```
prefix="/usr/local"
libdir="lib"
includedir="include"

Name: libgit2
Description: The git library, take 2
Version: 0.99.0
Libs: -L${libdir} -lgit2
Libs.private: -lrt -lpthread -lssh2
Requires.private: openssl libpcre zlib
Cflags: -I${includedir} 
```

Note that this means it produces `-Llib` and `-Iinclude` instead of the combined paths.

### Version of libgit2 (release number or SHA1)

be36db28f64de8ed5ca9870708a56b6668c6c482.

### Operating system(s) tested

Gentoo Linux

**Comment by mgorny** [comment]
My suggestion would be to use standard CMake `GNUInstallDirs` module instead of reinventing the wheel. It automatically provides both relative paths and absolute paths, the latter suitable for using in generated `.pc` files.

**Comment by mgorny** [comment]
Original bug report: https://bugs.gentoo.org/712270

**Comment by pks-t** [comment]
Thanks a lot for your bug report! Could you please verify that #5455 fixes the problem?

**Comment by pks-t** [comment]
Closed via #5455. Please feel free to reopen in case you feel like the error wasn't addressed properly. And thanks again for your report!
